.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  ci:
+    runs-on: ubuntu-20.04 # 22.04 doesn't have Python 3.6
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
We should probably drop support for 3.6; I don't see a need to support it for future versions.

Reply:
Sure, I can rip that out :)

Reply:
Done